src/pudl/output/ferc1.py

```diff
@@ -1337,3 +1340,205 @@ def remove_intra_table_calculated_values(exploded: pd.DataFrame) -> pd.DataFrame
         )
     ]
     return exploded
+
+
+################################################################################
+# XBRL Calculation Tree
+################################################################################
+class Ferc1XbrlCalculationTree(BaseModel):
```

Comment:
i find this name a lil too ambiguous. it took me a minute to realize this was a tree for a *specific* `xbrl_factoid`. Also the putting `Ferc1` first feels the opposite of our general convention (mostly, there are a few instances of this in transform.ferc1)